package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ak-url-validate",
+  "description": "[![Build Status](https://api.travis-ci.com/arshadkazmi42/ak-url-valdiate.svg?branch=master)](https://api.travis-ci.com/arshadkazmi42/ak-url-valdiate)",
```

Comment:
name of package

Reply:
I have removed description. We will add the description when we are getting this read for NPM release